src/css/docs-prose.css

```diff
@@ -39,23 +21,13 @@
   color: #3151df !important;
 }
 
-/* improve contrast for color-gray on bg-gray-faint */
```

Comment:
@katydecorah I removed this because the new gray in v1 looks a bit darker without this color override. Is there a standard way you test color contrast ratios?